doc/Learn/Maui/MauiOverview.md

```diff
@@ -34,7 +34,9 @@ For new applications created using the Uno Platform Template Wizard for Visual S
 
     ![Project structure](Assets/GettingStarted04-ProjectStructure.png)
 
-1. The application can be run by setting the desired platform project (Mobile or Windows) to be the startup project and then pressing F5 or clicking the Run button. The default layout shows a text, "Hello Uno Platform", which is an Uno TextBlock, followed by an Image, two TextBlock and, a Button, which are all .NET MAUI controls.
+1. The application can be run by selecting the desired platform in the dropdown, then pressing F5 or clicking the Run button. The default layout shows a text, "Hello Uno Platform", which is an Uno TextBlock, followed by an Image, two TextBlock and, a Button, which are all .NET MAUI controls.
```

Comment:
@jeromelaban maybe we can re-use some of the content from here: **get-started - create an app - debug the app**
https://github.com/unoplatform/uno/blob/d32f360003ff9365b359d55ec8fccf198a31041a/doc/articles/create-an-app-vs2022.md?plain=1#L36

To be able to have all the details about how to run the app for each platform (of course here in this file it would be tabs with only supported platforms for Maui embedding (iOS, Android, MacCatalyst, and Windows).
To re-use the debug details at different places like here for extensions or for themes, toolkit, figma,... doc.